webapp/components/Editor/Editor.spec.js

```diff
@@ -88,6 +91,29 @@ describe('Editor.vue', () => {
         )
       })
 
+      describe('limists suggestion list to 15 users', () => {
+        beforeEach(() => {
+          let manyUsersList = []
+          for (let i = 0; i < 25; i++) {
+            manyUsersList.push({ id: `user${i}` })
+          }
+          propsData.users = manyUsersList
+          wrapper = Wrapper()
+        })
+
+        it('when query is empty', () => {
+          expect(
+            wrapper.vm.editor.extensions.options.mention.onFilter(propsData.users),
+          ).toHaveLength(15)
+        })
+
+        it('when query is present', () => {
+          expect(
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-02T12:47:36Z" title="Monday, September 2nd 2019, 2:47:36 pm +02:00">Sep 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -98,6 +111,16 @@ describe('Editor.vue', () => {
         expect(wrapper.vm.editor.extensions.options.hashtag.items()).toEqual(propsData.hashtags)
       })
 
+      it('limits the suggestion list to 15 users', () => {
```


copy pasta! 🍝 😆 should be `hashtags` instead of `users` here...

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-02T13:49:54Z" title="Monday, September 2nd 2019, 3:49:54 pm +02:00">Sep 2, 2019</time>_

---


you caught me :open_mouth: